### PR TITLE
Merge speaker talk data file

### LINKED ISSUE #7: Merge speaker and talk data files
Make it easier for organisers by only having one data file (possibly new content type instead?) to fill in about speakers and the talk they're going to give.

- [ ] Make a single data template
- [ ] Adjust speaker partial to iterate over relevant fields in new location
- [ ] Adjust talk partial to iterate over relevant fields in new location

### PR DESCRIPTION
**Pull request by RMHogervorst**
Fix #7 and probably #28 too.

### CODE REVIEW

**Comment by RMHogervorst** [comment]
Modify the modal to also include bio of speaker and uses the profile picture:
Screenshot 2020-10-20 at 08 31 16
Talks part still works, tweetthis partial includes name of speaker (easy because it is in the same file)
Screenshot 2020-10-20 at 08 30 35

Screenshot 2020-10-20 at 08 30 52

**Comment by RMHogervorst** [comment]
Summarizing:

This PR:
* merges the speaker and talks files into one schedule/TalkXX.toml 
* adds that file to the theme
* modifies the relevant partials
* rebased against develop

**Comment by DaveParr** [comment]
This looks really great, and thanks for the rebase, you'd read my mind. I've decided (partly thanks to all your work this month) that we need to have a develop branch as the PR target. 

I've changed the target of this PR to that manually, I think because of your rebase that means everything is as intended.

I'm very happy to merge this in right now, and it does close #7 . It also almsot completely closes #28 with one exception:

The twitter user is not tagged by their handle (e.g. @sometwitterthing ) in the intent. I think this should be doable with a 'small' modification:

Currently the `social` section of what is now the `data/schedule/talkx.toml` like it does [in this file from Columbus2020](https://github.com/satRdays/Columbus2020/blob/master/data/speakers/alec-wong.toml). The key/awakward part is that it expects a full string such as `"https://twitter.com/Awong234"`. Would it be possible to extend the `tweetthis.html` partial with some form of string parsing, so it extracts only the handle from the url?

The double awkward bit would be the logic of putting in only the speakers 'listed' name if that twitter handle value was not present.

If you're interested in having a crack at that it would be awesome. As it is right now though I'd gratefully merge this in as is if you'd rather.

File: layouts/index.html [hunk 1]
```diff
@@ -76,4 +80,8 @@
   {{ partial "js.html" . }}
 </body>
 
-</html>
\ No newline at end of file
+<<<<<<< HEAD
+</html>
+=======
+</html>
+>>>>>>> 8ae97e6... include hackathon section
```
Comment (DaveParr) [inline layouts/index.html:87]: Something might have gone a little awry here with a git conflict. Effectively this change just isn't a change :)

**Review by DaveParr — CHANGES_REQUESTED** [review]
Sorry, tiny change needed here. Git conflict seemed to go a bit wonky.

**Comment by RMHogervorst** [comment]
Let me check tomorrow

**Comment by RMHogervorst** [comment]
I think that string parsing thing should to be a separate issue.
Let me check if I can fix it up today.

**Review by DaveParr — APPROVED** [review]

**Comment by DaveParr** [comment]
I'm going ahead to merge this. We can tackle that twitter handle string parsing problem in another branch.

Thanks a tonne for this :)

**Pull request merged**